swarm_copy/app/database/sql_schemas.py

```diff
@@ -0,0 +1,48 @@
+"""Schemas for the chatbot."""
+
+import datetime
+import uuid
+
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String
+from sqlalchemy.orm import declarative_base, relationship
+
+Base = declarative_base()
```

Comment:
This syntax is also not recommended anymore. One should be using the subclassing approach.

See https://docs.sqlalchemy.org/en/20/orm/declarative_styles.html#using-a-declarative-base-class

Reply:
<img width="701" alt="Screenshot 2024-10-31 at 11 45 22" src="https://github.com/user-attachments/assets/fe0d4d28-ecaf-4763-b9cc-81a2b6b07045">

Reply:
fixed in e60ad53